server/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Duane Johnson",
   "license": "GPL-3.0-only",
   "scripts": {
-    "start": "cross-env TMP_DIR=data/tmp ASSETS_DIR=data/assets DATABASE_HOST=localhost node --inspect src/server.js",
+    "start": "cross-env TMP_DIR=data/tmp ASSETS_DIR=data/assets JWTSECRET=jwt-secret-key DATABASE_HOST=localhost node --inspect src/server.js",
```

Comment:
Is there a way we could keep the JWTSECRET completely optional here? For example, so that I don't have to set JWTSECRET in the relm.us production environment? (This isn't a big deal, but I'm thinking it would be nice for newcomers to have to set only a few environment variables, i.e. for features they know they need.)